pillar/params.sls

```diff
@@ -172,3 +172,6 @@ ldap:
   admin_group_name: ''
   tls_method: ''
   mail_attribute: ''
+
+# infra container to use instead of downloading gcr.io/google_containers/pause
+pod_infra_container_image: sles12/pause:1.0.0
```

Comment:
I wonder if that should be `docker.io/sles12/pause:1.0.0` for crio, but apparently it doesn't make a difference

Reply:
Currently, it's working because we're configuring `/etc/containers/registries.conf` to search in `docker.io` when using unqualified images. It would be safer to always use fully qualified images, but both ways are fine for me atm.

Reply:
I will leave the code as it is, to avoid clutter inside of the salt states.